ui/app/styles/core/buttons.scss

```
@@ -327,3 +327,14 @@ a.button.disabled {
   font-weight: inherit;
   cursor: pointer;
 }
+
+// Modification to HDS copy button component to match current UI
+.hds-copy-button {
+  &.icon-only {
+    background: none;
+    border: none;
+    box-shadow: none;
+    min-width: $spacing-l;
+    padding: 0;
+  }
+}
```

So far I have only implemented icon-only HDS copy buttons. So this could be changed to the suggestion below and we could just use one class. But I made the class like this instead for now, in case there will be non icon-only copy buttons and we need some overarching/shared styles. I can change it to the suggestion if I never end up using non icon-only copy buttons
```suggestion
.hds-copy-button {
  background: none;
  border: none;
  box-shadow: none;
  min-width: $spacing-l;
  padding: 0;
}
```